prime-router/metadata/fhir_mapping/hl7/datatypes/CNE/CodeableConcept.yml

```diff
@@ -0,0 +1,115 @@
+coding:
+  generateList: true
+  expressionType: nested
+  vars:
+    code: String, CNE.1
+    display: String, CNE.2
+    system: SYSTEM_URL, CNE.3
+    version: String, CNE.7
+  expressionsMap:
+    code:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.1
+    display:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.2
+    system:
+      type: SYSTEM_URL
+      expressionType: HL7Spec
+      valueOf: CNE.3
+    version:
+      type: String
+      expressionType: HL7Spec
+      valueOf: CNE.7
+    extension_CNE-coding:
+      condition: $code NOT_NULL || $display NOT_NULL || $system NOT_NULL || $version NOT_NULL
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: CNE-coding
+        valueString:
+          value: coding
+    extension_CNE-coding-system:
+      condition: $coding-system NOT_NULL
+      generateList: true
+      expressionType: nested
+      vars:
+        coding-system: String, CNE.3
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: CNE-coding-system
+        valueString:
+          value: $coding-system
+
+coding_alt:
```

Comment:
are we handling the second alt in this mapping?

Reply:
I'll add the second alt, but FYI it's not in the NIST spec though it does exist in the Java class